docs/source/cli_help/discover.rst

```diff
@@ -13,6 +13,7 @@ Discovers and displays tree of branch dependencies using a heuristic based on re
 If confirmed with a ``y[es]`` or ``e[dit]`` reply, backs up the current definition file (if it exists) as ``$GIT_DIR/machete~``
 and saves the new tree under the usual ``$GIT_DIR/machete`` path.
 If the reply was ``e[dit]``, additionally an editor is opened (as in: ``git machete`` :ref:`edit`) after saving the new definition file.
+Cannot modify existing ``git machete traverse`` branch annotation qualifiers.
```

Comment:
The subject (aka _podmiot_) of the sentence isn't obvious... I'd state it explicitly:
```suggestion
If executed when a definition file already exists, `discover` retains the existing ``git machete traverse`` branch annotation qualifiers.
```

Also... what's the naming? `branch annotation qualifiers`? I'd just call it `branch qualifiers` everywhere for consistency 🤔